lib/omniscrapper_hq/operations/event/index.rb

```diff
@@ -0,0 +1,23 @@
+module Operations
+  module Event
+    class Index < Base
+      GRAPHQL_CLIENT = System::Container['omniscrapper.graphql_client']
+
+      Query = GRAPHQL_CLIENT.parse <<-'GRAPHQL'
+        query {
+          events {
+            id,
+            name,
+            frequencyPeriod,
+            frequencyQuantity
+          }
+        }
+      GRAPHQL
+
+
+      def call
+        GRAPHQL_CLIENT.query(Query)
+      end
+    end
+  end
+end
```

Comment:
![image](https://user-images.githubusercontent.com/26395360/68724432-76340300-0589-11ea-8d9a-14f9f2e6ea73.png)

These lines causing this warning. Not sure how to fit it. Especially when we need static constant as it is required by graphql-client https://github.com/github/graphql-client/blob/master/guides/dynamic-query-error.md